twill-yarn/src/main/java/org/apache/twill/internal/ServiceMain.java

```diff
@@ -264,6 +262,10 @@ protected String getLoggerLevel(Logger logger) {
     return "OFF";
   }
 
+  protected String getLogLevels() {
+    return "";
```

Comment:
Why we can't set the custom log levels at launch in the same programmatic way as when a system message is received?